File: src/test/java/org/goafabric/catalog/controller/InsuranceControllerIT.java
package org.goafabric.catalog.controller;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import static org.assertj.core.api.Assertions.assertThat;

@SpringBootTest
public class InsuranceControllerIT {
    @Autowired
    private InsuranceController insuranceController;

    @Test
    void findByCode() {
        assertThat(insuranceController.findByCode("168140040")).isNotNull();
    }

    @Test
    void search() {
        assertThat(insuranceController.findByDisplay("")).isNotNull().isNotEmpty();
    }
}

File: src/test/kotlin/org/goafabric/catalog/controller/InsuranceControllerIT.kt
package org.goafabric.catalog.controller

import org.assertj.core.api.Assertions
import org.goafabric.catalog.job.insurance.InsuranceEo
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
internal class InsuranceControllerIT {
    @Autowired
    private val insuranceController: InsuranceController? = null

    @Test
    fun findByCode() {
        Assertions.assertThat<InsuranceEo>(insuranceController!!.findByCode("168140040")).isNotNull()
    }

    @Test
    fun search() {
        Assertions.assertThat<InsuranceEo?>(insuranceController!!.findByDisplay("")).isNotNull().isNotEmpty()
    }
}
